hack/istio/install-bookinfo-demo.sh

```diff
@@ -211,15 +224,35 @@ $CLIENT_EXE get services -n ${NAMESPACE}
 $CLIENT_EXE get pods -n ${NAMESPACE}
 
 # If OpenShift, we need to do some additional things
-if [[ "$CLIENT_EXE" = *"oc" ]]; then
+if [[ "${IS_OPENSHIFT}" = "true" ]]; then
```

Comment:
use single-brackets and double-equals:
```suggestion
if [ "${IS_OPENSHIFT}" == "true" ]; then
```